src/components/button-toggle/button-toggle.style.js

```diff
@@ -0,0 +1,81 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { THEMES } from '../../style/themes';
+
+const StyledButtonToogle = styled.div`
+  display: inline;
+`;
+
+const StyledButtonToogleLabel = styled.label`
+  display: inline-block;
+  height: 40px;
+  padding: 0 24px;
+  border: 1px solid ${({ theme }) => theme.colors.border};
+  font-size: 14px;
+  font-weight: 600;
+
+  input:checked ~ & {
+    color: ${({ theme }) => theme.colors.white};
+    background-color: ${({ theme }) => theme.colors.tertiary};
+  }
+
+  &:hover {
+    background-color: ${({ theme }) => theme.colors.whiteMix};
+    border-color: ${({ theme }) => theme.colors.tertiary};
+  }
+
+  &:focus {
+    outline: 3px solid ${({ theme }) => theme.colors.focus};
+  }
+
+  .content-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+  }
+
+  ${({ buttonIconSize }) => buttonIconSize === 'large' && css`
+    min-width: 104px;
+    height: 104px;
+    padding: 0 16px;
+
+    .content-wrapper {
+      flex-direction: column;
+    }
+  `}
+`;
+
+const StyledButtonToggleIcon = styled.div`
+  ${({ buttonIconSize }) => buttonIconSize === 'large' && css`
+    .carbon-icon::before {
+      font-size: 40px;
+      line-height: 40px;
+    }
+    .carbon-icon {
+      margin-bottom: 8px;
+    }
+  `}
+
+  ${({ theme }) => theme.name === THEMES.classic && css`
+    .carbon-icon::before {
+      font-size: 60px;
```

Comment:
![Zrzut ekranu 2019-04-18 o 09 19 00](https://user-images.githubusercontent.com/32877259/56343479-3061b880-61bb-11e9-9794-8fdb615d4ef9.png)
why is icon size 60px?